app/routes/how-can-i-help.knowledge.tsx

```diff
@@ -109,7 +202,15 @@ const DiveDeeper = () => (
             action: 'https://aisafetyfundamentals.com/alignment/',
           },
         ]}
-        // TODO: add "full list of courses"
+        additionalInfo={
+          <>
+            Or, browse our{' '}
+            <a href="https://www.aisafety.com/courses" className="small-bold teal-500">
+              full list of courses
+            </a>
+          </>
+        }
+        // TODO: fix additionalInfo appearing for each link
```

Comment:
I'm not sure what you mean by this, maybe merge and then I'll see

Reply:
You can see that the "Or, browse etc." appears twice, once for each link